content/doc/administration/requirements/java.adoc

```diff
@@ -9,11 +9,12 @@ There are separate run and job execution requirements for Jenkins installations.
 
 Modern Jenkins versions have the following Java requirements:
 
-* Java 8 is the **ONLY** currently supported runtime environment, both 32-bit and 64-bit versions are supported
+* Java 8 runtime environments, both 32-bit and 64-bit versions are supported
+* Since Jenkins `2.155`, Java 11 runtime environments is supported
```

Comment:
I would put 2.163, and use a `footnote:[]` if we want to explain details why 2.155 was actually the first one. So in case users read this too quickly, they don't end up with 2.155.

Reply:
Noted, will do.

Reply:
I did that.